model/messages/collection.go

```diff
@@ -41,10 +41,11 @@ type ClusterBlockVote struct {
 }
 
 // ClusterTimeoutObject is part of the collection cluster protocol and represents a collection node
-// timing out in given round.
+// timing out in given round. Contains a sequential number for deduplication purposes.
 type ClusterTimeoutObject struct {
```

Comment:
I like [Jordan's suggestion](https://github.com/onflow/flow-go/pull/3430/files#r1001453978) to define the cluster types through the consensus types:
```golang
// ...
type ClusterTimeoutObject TimeoutObject

// ...
type ClusterBlockVote BlockVote
```